packages/gallery/src/components/gallery/proGallery/galleryContainer.js

```diff
@@ -704,6 +714,15 @@ export class GalleryContainer extends React.Component {
     if (typeof this.props.eventsListener === 'function') {
       this.props.eventsListener(eventName, eventData, event);
     }
+
+    if (eventName === GALLERY_CONSTS.events.GALLERY_SCROLLED) {
+      const { top, left } = eventData;
+      this.videoScrollHelper.trigger.SCROLL({
+        top,
+        left,
+      });
+      this.setScrollTop(top)
```

Comment:
set scroll state. the scroll state is an object of {top, left}
Functions can use either one fut in these functions we must update on what happened. not what we want to use right now.